Packs/CommonPlaybooks/Playbooks/playbook-Domain_Enrichment_-_Generic_v2.yml

```diff
@@ -395,22 +399,91 @@ inputs:
 outputs:
 - contextPath: Domain
   description: The domain objects.
-  type: unknown
+  type: string
 - contextPath: DBotScore
   description: Indicator, Score, Type, and Vendor.
-  type: unknown
+  type: string
 - contextPath: Domain.Name
   description: Bad domain found.
+  type: string
 - contextPath: Domain.Malicious.Vendor
   description: For malicious domains, the vendor that made the decision.
+  type: string
 - contextPath: DBotScore.Indicator
   description: The indicator that was tested.
+  type: string
 - contextPath: DBotScore.Type
   description: The indicator type.
+  type: string
 - contextPath: DBotScore.Score
   description: The actual DBot score.
-- contextPath: DBotScore.Reliability
```

Comment:
We no longer have reliability to output?

Reply:
No since its not part of any of the command outputs.